### PR TITLE
Revert "chore(query): introduce meta Runtime"

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Reverts datafuselabs/databend#5294

## Changelog

- Build/Testing/CI

### CODE REVIEW

File: .github/workflows/developing.yml [hunk 1]
```diff
@@ -83,12 +83,13 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/test_stateless_standalone_linux
 
-  test_stateless_standalone_macos:
-    runs-on: macos-11
-    needs: build_macos
-    steps:
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/test_stateless_standalone_macos
+  # https://github.com/datafuselabs/databend/issues/5287
```
Comment (Xuanwo) [inline .github/workflows/developing.yml:86]: Please also comment the same action in `production.yml`

**Comment by Xuanwo** [comment]
I have updated the description to make our friend fusebot happy.

**Review by Xuanwo — APPROVED** [review]

**Comment by BohuTANG** [comment]
The revert reason is the `meta-runtime` not work?

**Comment by sundy-li** [comment]
> The revert reason is the `meta-runtime` does not work?

It works, but it may make the runtime blocked in this sync call. Searching for other ways.

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**